src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/alerts-classic-vs-workflows.mdx

```diff
@@ -0,0 +1,912 @@
+---
+title: Notification migration 
+metaDescription: Migrating notifications from alerts and applied intelligence 
+redirects: 
+---
+
+At the end of January 2023, the classic notification system for alerts and applied intelligence will be deprecated and we will be implementing the use of workflows instead. This is good news because workflows is an improved, flexible notification system that helps your team learn about potential errors faster so you can take immediate action.
```

Comment:
Let's drop the exact date from the opening.  It should probably move it to a new section about the automated migration above the template.
```suggestion
The classic notification system for alerts and applied intelligence is being replaced by workflows. This is good news because workflows is an improved, flexible notification system that helps your team learn about potential errors faster so you can take immediate action.
```

That second sentence may need to be reworked to emphasize the flexibility, number of integrations and correlations across issues, since technically workflows is slower due to the grace period to try and correlate more.